data-model/src/main/avro/RawSpan.avdl

```diff
@@ -12,6 +13,8 @@ protocol RawSpanProtocol {
 
     array<Entity> entity_list = [];
 
+    union { null, Resource } resource = null;
```

Comment:
Because span normalization right now works one span at a time (Ref: https://github.com/hypertrace/hypertrace-ingester/blob/296351d7b6b7b89061cf581f69c7fa0da83f46bf/span-normalizer/span-normalizer/src/main/java/org/hypertrace/core/spannormalizer/jaeger/JaegerSpanToAvroRawSpanTransformer.java#L43 ) we're stuck flattening resources across each RawSpan at the moment. The StructuredTrace data model re-normalizes it so this limitation does not cause all spans later in the pipeline to bloat.